copyrightify/copyrightify.py

```diff
@@ -168,7 +168,7 @@ def process_paths(paths, recursive, config, context):
 
 def main():
     try:
-        config = yaml.load(resource_string(__name__, 'config.yaml'))
+        config = yaml.load(resource_string(__name__, 'config.yaml'), Loader=yaml.FullLoader)
```

Comment:
I think it is generally recommended to use the `safe_load` command instead of the `load` command.
https://security.openstack.org/guidelines/dg_avoid-dangerous-input-parsing-libraries.html
Which I think is basically the equivalent of `Loader=yaml.SafeLoader`.